kornia/testing/__init__.py

```diff
@@ -146,9 +145,9 @@ def assert_close(
                 This parameter allows to reduce tolerance. Half the decimal places.
                 Example, 1e-4 -> 1e-2 or 1e-6 -> 1e-3
         """
-        if isinstance(actual, Parameter):
+        if hasattr(actual, "data"):
```

Comment:
`has_attr` is bad for JIT. I know that assert_close is for testing, so just FYI

Reply:
yeah, just realizing how evil is torchscript